internal/plugin/plugin.go

```diff
@@ -19,10 +19,12 @@ import (
 const (
 	devboxDirName       = "devbox.d"
 	devboxHiddenDirName = ".devbox"
-	VirtenvBinPath      = ".devbox/virtenv/bin"
 	VirtenvPath         = ".devbox/virtenv"
 )
 
+var WrapperPath = filepath.Join(VirtenvPath, ".wrappers")
+var WrapperBinPath = filepath.Join(WrapperPath, "bin")
```

Comment:
I plan on moving this out of here, but I just wanted to get this PR out quickly because we're not sure how many people this is affecting.